### PR TITLE
[SF 4.0] Make actions explicitly public

### PR DESCRIPTION
**Pull request by GuilhemN**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

As of sf 4.0, controllers must be explicitly made public.

### CODE REVIEW

**Comment by dunglas** [comment]
Can you target 2.1 as it's a bug fix?

**Comment by GuilhemN** [comment]
@dunglas done ;)

File: src/Bridge/Symfony/Bundle/Resources/config/api.xml [hunk 1]
```diff
@@ -168,27 +168,27 @@
...
-        <service id="api_platform.action.get_collection" alias="api_platform.action.placeholder" />
-        <service id="api_platform.action.post_collection" alias="api_platform.action.placeholder" />
-        <service id="api_platform.action.get_item" alias="api_platform.action.placeholder" />
-        <service id="api_platform.action.put_item" alias="api_platform.action.placeholder" />
-        <service id="api_platform.action.delete_item" alias="api_platform.action.placeholder" />
-        <service id="api_platform.action.get_subresource" alias="api_platform.action.placeholder" />
-
-        <service id="api_platform.action.entrypoint" class="ApiPlatform\Core\Action\EntrypointAction">
+        <service id="api_platform.action.placeholder" class="ApiPlatform\Core\Action\PlaceholderAction" public="true" />
+        <service id="api_platform.action.get_collection" alias="api_platform.action.placeholder" public="true" />
```
Comment (sroze) [inline src/Bridge/Symfony/Bundle/Resources/config/api.xml]: Do we really need these aliases to be public? 🤔

Comment (GuilhemN) [reply src/Bridge/Symfony/Bundle/Resources/config/api.xml]: If they're used in routing, yes. At least they were public until now so I think they should still be when using sf 4.0.

**Comment by dunglas** [comment]
Thanks @GuilhemN

**Pull request merged**